mayastor/src/grpc/mod.rs

```diff
@@ -40,6 +40,9 @@ mod json_grpc;
 mod mayastor_grpc;
 mod nexus_grpc;
 mod server;
+pub mod mayastor {
```

Comment:
do we really need the outer `mayastor` mod? We only have 1 API here, even if there could be multiple versions of it.

Reply:
The mayastor mod is the folder that is introduced inside the grpc folder which will have the v1, v2 etc. Also we put all the mods into one we may end up conflicting structs and methods as all will be having the same names

Reply:
Does the mayastor mod need to be public? Only the v1 is needed to be pub right?